src/actions/integrations.ts

```diff
@@ -387,43 +315,31 @@ export function getAuthorizedOAuthApps(): ActionFunc {
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
 
-            dispatch(batchActions([
-                {type: IntegrationTypes.GET_AUTHORIZED_OAUTH_APPS_FAILURE, error},
-                logError(error),
-            ]), getState);
+            dispatch(logError(error));
 
             return {error};
         }
 
-        dispatch({type: IntegrationTypes.GET_AUTHORIZED_OAUTH_APPS_SUCCESS, data: {}});
-
         return {data};
     };
 }
 
 export function deauthorizeOAuthApp(clientId: string): ActionFunc {
     return bindClientFunc({
         clientFunc: Client4.deauthorizeOAuthApp,
-        onRequest: IntegrationTypes.DEAUTHORIZE_OAUTH_APP_REQUEST,
         onSuccess: IntegrationTypes.DEAUTHORIZE_OAUTH_APP_SUCCESS,
```

Comment:
You don't need the `onSuccess` here, the `DEAUTHORIZE_OAUTH_APP_SUCCESS` is not used anywhere (I think).